src/executors/java_builder_docker.yml

```diff
@@ -0,0 +1,20 @@
+description: >
+  Docker executor with JDK to build java applications.
+
+parameters:
+  java_version:
+    description: "Java version to install (or rather selecting an existing image accordingly)"
+    type: string
+    default: "11.0"
+  docker_hub_username:
+    type: string
+    default: $DOCKER_JFROG_USERNAME
+  docker_hub_password:
+    type: string
+    default: $DOCKER_JFROG_PASSWORD
```

Comment:
`cimg/openjdk` is stored on docker hub, not on JFrog, so you need docker credentials.